server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoVisitor.java

```diff
@@ -121,7 +123,11 @@ private void addReplacementContext(ParserRuleContext ctx) {
     String newText = context.getExtendedDocument().toString()
             .substring(ctx.start.getStartIndex(), ctx.stop.getStopIndex() + 1)
             .replaceAll("[^ \n]", CobolDialect.FILLER);
-    Range range = DialectUtils.constructRange(ctx);
+    Token start = ctx.start;
+    Token stop = ctx.stop;
+    Range range = new Range(new Position(start.getLine() - 1, start.getCharPositionInLine()),
+            new Position(stop.getLine() - 1,
+                    stop.getCharPositionInLine() + stop.getStopIndex() - stop.getStartIndex()));
```

Comment:
Should this be the same?
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsVisitor.java#L196-L199

Reply:
It should be fine, daco grammar doesn't consume dots.

Reply:
Oops, there are dots expected.